src/land/ArcanistSubmitQueueEngine.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+final class ArcanistSubmitQueueEngine
```

Comment:
Rename file and class to `UberArcanistSubmitQueueEngine`. We keep this agreement to identify which code parts were created by us.